src/main/io/osd.c

```diff
@@ -1127,6 +1131,12 @@ STATIC_UNIT_TESTED void osdRefresh(timeUs_t currentTimeUs)
 {
     static timeUs_t lastTimeUs = 0;
 
+#ifdef USE_OSD_ADJUSTMENTS
+    if (isAnyAdjustmentFunctionBusy() != 0) {
```

Comment:
This function is `bool` so just use `if (isAnyAdjustmentFunctionBusy()) {`

Reply:
omg. fixed that.